todo_app/templates/index.html

```diff
@@ -12,13 +12,52 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
-        {%for items in items%}<!--Used flask notation to loop through the items list-->
-          <li>{{ items.id }} - {{ items.status }} - {{ items.title }}</li><!--Used flask notation to add items into the unordered list-->
+        {%for card in cards%}
+          {% if card.idList == doing %}
+            <li>{{ card.name }} - {{ card.desc }} - {{ card.due }} - Doing 
+              <form action="/todo" method="post">
+                <input type="text" name="id" value="{{card.id}}" hidden>
+                <button type="submit">To Do</button>
+              </form>
+              <form action="/done" method="post">
+                <input type="text" name="id" value="{{card.id}}" hidden>
+                <button type="submit">Done</button>
+              </form>
+            </li>
+          {% endif %}
+          {% if card.idList == to_do %}
+            <li>{{ card.id.hidden }}{{ card.name }} - {{ card.desc }} - {{ card.due }} - To-Do 
```

Comment:
What's the intention of `{{ card.id.hidden }}`?
```suggestion
            <li>{{ card.name }} - {{ card.desc }} - {{ card.due }} - To-Do 
```

Same for "done" cards.